source/features/show-whitespaces.tsx

```diff
@@ -0,0 +1,96 @@
+import './show-whitespaces.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import onPrFileLoad from '../libs/on-pr-file-load';
+import onNewComments from '../libs/on-new-comments';
+import getTextNodes from '../libs/get-text-nodes';
+
+function showWhiteSpacesOn(line: Element): void {
```

Comment:
Could you add a test for this function? Nothing too fancy, just ensure that it works on 4 lines like:


```
noIndent(' ' + ' ');
 spaceIndent('   double   ');
  doubleSpace;
	// tab
		// double tab
	 	// tab space tab
```

Reply:
Tried implementing tests (with various hacks for types inconsistency, and handling CSS imports), but stuck at this error.

```
  Uncaught exception in test/show-whitespace.ts

  /mnt/d/dev/git/sindresorhus/refined-github/source/options-storage.ts:20

   19: const featureOptions: Partial<RGHOptions> = {};
   20: for (const feature of __featuresList__) {
   21:   featureOptions[`feature:${feature}`] = true;

  ReferenceError: __featuresList__ is not defined
```

Reply:
When you resolve this I miss it

Reply:
If you want to test it, it'd have to be extracted into its own file because regular feature files depend on a lot of dangerous files like `features.tsx` and that webpack-replaced variable. Perhaps this function can be moved to `libs/show-whitespace.tsx`